### DISCUSSION TITLE
.bash_history integrate with gnome environment

### DISCUSSION THREAD

**Discussion by CaptainMorgan12** [lxqt/lxqt#2274] [General]
Does qterminal use the same .bash_history file in ~/ directory?

If not is there a way to share history between gnome-terminal and qterminal?

**Comment by tsujan** [comment]
> Does qterminal use the same .bash_history file in ~/ directory?

It does.

**Comment by darkshram** [comment]
It should, unless you use zsh, fish, ksh, tcsh or any other alternative shell as default for your user account. Files as ~/.bashrc, .bash_history, etc. are used by bash, so your user account needs to use bash as default shell. Terminals had nothing to do with this files. Terminals will use the default shell defined for your account.

look at https://linuxhandbook.com/shell-using/ to find out which shell is using your user account.
